packages/order/src/models/line-item-adjustment.ts

```diff
@@ -28,7 +22,7 @@ export default class LineItemAdjustment extends AdjustmentLine {
     entity: () => LineItem,
     columnType: "text",
     fieldName: "item_id",
-    cascade: [Cascade.REMOVE],
+    onDelete: "cascade",
```

Comment:
todo: I believe I haven't seen that change in the migration, could you double check :)

Reply:
the migration was correct already